### DISCUSSION TITLE
Rejected applications for no provided reason.

### DISCUSSION THREAD

**Discussion by CrypLemonade** [community/community#64991] [GitHub Education]
Hello, I’m a 15 year old boy who attends school in the United Kingdom.

I have attempted to re-verify several times but each and every time I have tried have been shut down with a rejection.

The proof of affiliation with my school clearly states my name, my schools name, my date of birth, date of admission and even my school email and login. 

Every time I receive a rejection it doesn’t even provide a reason or even submit it for that matter - I just get auto-rejected. I click submit, it reaches 100% and I get rejected. 

The rejection message just states this: 

“Unfortunately, we weren’t able to approve your educational discount request for CrypLemonade.”

“What happened? Common verification problems and suggested solutions are documented here: (some link)”

I’m confused as to why I’m being rejected without a proper reasoning especially considering my evidence of affiliation clearly shows I attend the school.

It would be deeply appreciated if someone would help me? 

Many Thanks,
 - Dylan

**Comment by arafajs** [comment]
1. Make sure that the proof of affiliation you are uploading is clear and legible. The document should show your name, your school's name, and the dates of your enrollment.
2. Try uploading a different document as proof of affiliation. You can use a school ID, a transcript, or a letter from your school.
3. Contact GitHub Education support for help. They may be able to provide more information about why your verification is being rejected.
4. Make sure that your GitHub username is your full name as it appears on your school ID or transcript.
5. Complete your GitHub user profile with a short bio and a link to your school's website.
6. If you are a distance learner, make sure to include that information in your application.
7. Be patient. It may take some time for GitHub Education to process your application.

**Reply by CrypLemonade** [reply]
Hey, I’ve tried Letters, School portal, school account information and I’m still getting rejected but they aren’t even providing me a reason it’s kinda annoying.

**Reply by arafajs** [reply]
@CrypLemonade As long as there is an automatic and quick rejection, it seems that the matter is related to the basic information of your student ID. Age is often the reason.

**Reply by CrypLemonade** [reply]
I think it might be my GitHub or something because I resubmitted again and this one actually went through
